spec/presenters/two_factor_authentication/set_up_piv_cac_selection_presenter_spec.rb

```diff
@@ -38,13 +38,13 @@
     context 'with a confirmed email address ending in anything other than .gov or .mil' do
       let(:user) { create(:user, email: 'example@example.com') }
 
-      it { expect(recommended).to eq(false) }
+      it { expect(recommended).to be_falsey }
```

Comment:
I'm not sure I'd expect this change, and I'd expect a predicate `?` method to always return a boolean, not `nil`. Can we leave this as it was, and coerce the `nil` value to `false` in the method if that's what we're expecting?

Reply:
it should return false. I just never know whether its better to use 'be_falsey' vs eq (false)

Reply:
I prefer `eq(false)` since it's more strict in expecting a value that's explicitly `false`. `be_falsey` would allow a value that's either `nil` or `false` ([ref](https://www.rubydoc.info/github/rspec/rspec-expectations/RSpec%2FMatchers:be_falsey)).